### PR TITLE
Update Account Protection API URIs

### PR DESCRIPTION
**Pull request by shrutichidam**

### CODE REVIEW

**Comment by chadsowald** [comment]
@shrutichidam did you get to try this locally?

**Comment by chadsowald** [comment]
@shrutichidam Did you happen to search the sample docs (/docs) to see if any of the docs have this URL in them?

**Comment by shrutichidam** [comment]
@chadoswald I haven't tried it locally yet. I did search the sample docs but didn't update those - I'll go ahead and do that as well.

File: docs/Configure the sample site.md [hunk 1]
```diff
@@ -67,8 +67,8 @@ Follow these steps to configure the sample site before running it.
       "Signup": "/v1.0/MerchantServices/events/Signup",
       "SignupStatus": "/v1.0/MerchantServices/events/SignUpStatus",
       "UpdateAccount": "/v1.0/MerchantServices/events/UpdateAccount",
-      "SignInAP": "/v0.5/MerchantServices/AccountProtection/events/{0}/AccountLogin/{1}",
-      "SignupAP": "/v0.5/MerchantServices/AccountProtection/events/{0}/AccountCreation/{1}"
+      "SignInAP": "/v1.0/action/account/login/{0}",
+      "SignupAP": "/v1.0/action/account/create/{0}}"
```
Comment (chadsowald) [inline docs/Configure the sample site.md:71]: extra ending bracket on this line

**Pull request merged**